src/main/java/com/ekart/model/User.java

```diff
@@ -38,14 +41,19 @@
     )
     private Long id;
 
+    @NotNull
+    @Size(min = 1)
     @Column(name = "first_name")
     private String firstName;
 
     @Column(name = "last_name")
     private String lastName;
 
+    @NotNull
     private byte[] password;
 
+    @NotNull
+    @CheckEmail
```

Comment:
Try using @Pattern annotation. If doesn't work then u can make the custom validator generic so that it can be reused for other fields aswell

Reply:
Generic pattern is present in email validator file. I have created @CheckEmail annotation so that it can be used in other files as well. Check Email uses email validator pattern for validating..

Reply:
will address in next pr